src/DynamoCore/Models/DynamoModelEvents.cs

```diff
@@ -203,6 +203,17 @@ public virtual void OnEvaluationCompleted(object sender, EvaluationCompletedEven
                 EvaluationCompleted(sender, e);
         }
 
+        /// <summary>
+        /// An event triggered when all tasks in scheduler are completed.
+        /// </summary>
+        public event Action<HomeWorkspaceModel> RefreshCompleted;
```

Comment:
Yes this is correct. :)

On Fri, Jun 5, 2015, 2:42 AM pbidenko <notifications@github.com> wrote:

> In src/DynamoCore/Models/DynamoModelEvents.cs
> <https://github.com/DynamoDS/Dynamo/pull/4634#discussion_r31799737>:
>
> > @@ -203,6 +203,17 @@ public virtual void OnEvaluationCompleted(object sender, EvaluationCompletedEven
> >                  EvaluationCompleted(sender, e);
> >          }
> >
> > +        /// <summary>
> > +        /// An event triggered when all tasks in scheduler are completed.
> > +        /// </summary>
> > +        public event Action<HomeWorkspaceModel> RefreshCompleted;
>
> Hi @pboyer <https://github.com/pboyer> ,
> I'm not sure if it is exactly what you want me to do... please, take a
> look.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/DynamoDS/Dynamo/pull/4634/files#r31799737>.
>